mozilla-release/config/version_win.pl

```diff
@@ -274,11 +274,11 @@ sub getNextEntry
 
 my $versionlevel=0;
 my $insideversion=0;
-if (open(RCINCLUDE, "<$rcinclude")) 
+if (open(RCINCLUDE, "<$rcinclude"))
 {
 	print RCFILE "// From included resource $rcinclude\n";
 #	my $mstring="";
-	while (<RCINCLUDE>) 
+	while (<RCINCLUDE>)
```

Comment:
I want to cut all fingers, one-by-one, for whom did this in this script :) Really, this is suck. My text editors configured to automatically remove trailing spaces. Also I'm prefer and suggest for all of us to use 2 spaces instead of tab where it possible (Makefile is problem here). So, let's decide how we want to live - with garbage in original FF or we will create our own better style?